pacman_contest_cluster.py

```diff
@@ -324,7 +333,7 @@ class ContestRunner:
     # submissions file format: s???????[_datetime].zip
     # submissions folder format: s???????[_datetime]
     # datetime in ISO8601 format:  https://en.wikipedia.org/wiki/ISO_8601
-    def __init__(self, organizer, teams_root, include_staff_team, staff_teams_dir, compress_logs,
+    def __init__(self, organizer, teams_root, staff_teams_vs_others_only, include_staff_team, staff_teams_dir, compress_logs,
```

Comment:
We also need to just pass here a whole dictionary called options, as it is unmanagable and error prone to have so many options in a method. Just opened #60 for it

Reply:
We actually do pass setting, so it is only a matter of re-formatting the formal parameters def